contrib/database/sql/conn.go

```diff
@@ -146,6 +146,13 @@ func (tc *tracedConn) CheckNamedValue(value *driver.NamedValue) error {
 	return driver.ErrSkip
 }
 
+func (tc *tracedConn) ResetSession(ctx context.Context) error {
```

Comment:
Can you please add a comment?
```suggestion
// ResetSession implements driver.SessionResetter
func (tc *tracedConn) ResetSession(ctx context.Context) error {
```